package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.59.1",
+      "version": "0.60.2",
```

Comment:
```suggestion
      "version": "0.59.1",
```

Reply:
I have commited the changes from here. Hope that it has been resolved.